mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -576,8 +609,7 @@ class DecisionTreeTest(sc: SparkContext) extends DecisionTreeTests(sc) {
     (splits, categoricalFeaturesInfo_, labelType)
   }
 
-  override def runTest(rdd: RDD[LabeledPoint])
-    : Either[RandomForestModel, GradientBoostedTreesModel] = {
+  override def runTest(rdd: RDD[LabeledPoint]): DecisionTreeModel = {
```

Comment:
Synced offline. Agreed that current implementation is OK since adding DataFrames to `createInputData` would be outside scope of this PR.